dotcom-rendering/src/components/DarkModeMessage.tsx

```diff
@@ -0,0 +1,28 @@
+import { css } from '@emotion/react';
+import { palette, space, textSans } from '@guardian/source-foundations';
+
+export const DarkModeMessage = () => (
+	<aside
+		css={css`
+			display: none;
+			@media (prefers-color-scheme: dark) {
+				padding: ${space[4]}px;
+				${textSans.medium()}
+				display: block;
+				background-color: ${palette.neutral[7]};
+				color: ${palette.neutral[97]};
+			}
+		`}
+	>
+		<p
+			css={css`
+				max-width: 60ch;
+			`}
+		>
+			As part of this beta release, we are currently lacking support for
+			“dark mode.” We hope to make it available in the coming weeks. In
```

Comment:
```suggestion
			“dark mode”. We hope to make it available in the coming weeks. In
```

Reply:
And to address @ddramowicz's very good points, perhaps the wording could be: 

As part of this beta release, some articles are currently lacking support for “dark mode”. We are working to fix this in the next beta release.

Reply:
[The style guide would have the punctuation inside, I believe](https://www.theguardian.com/guardian-observer-style-guide-q).

I’m not sure we need to “air quote” dark mode, though.

Reply:
@cemms1 correct!

I double-checked for an actual example with a phrase and found the following, where the comma is resolutely outside.

> **a capella**
> Italian for “in the style of the church”, ie unaccompanied singin